### PR TITLE
Fix and reintroduce arrival and departure dates

### PR DESCRIPTION
**Pull request by krzys-h**
(support in the participants table will be added by @grzegorz225 later)

### CODE REVIEW

File: static/js/moment-locale-pl.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+;(function (global, factory) {
+   typeof exports === 'object' && typeof module !== 'undefined'
+       && typeof require === 'function' ? factory(require('../moment')) :
+   typeof define === 'function' && define.amd ? define(['../moment'], factory) :
+   factory(global.moment)
+}(this, (function (moment) { 'use strict';
+
+
+    var monthsNominative = 'styczeń_luty_marzec_kwiecień_maj_czerwiec_lipiec_sierpień_wrzesień_październik_listopad_grudzień'.split('_'),
```
Comment (radrow) [inline static/js/moment-locale-pl.js]: Why isn't it a list of strings?

File: wwwapp/forms.py [hunk 2]
```diff
@@ -32,16 +33,44 @@ def __init__(self, *args, **kwargs):
...
+
     class Meta:
         model = UserInfo
-        fields = ['pesel', 'address', 'tshirt_size', 'comments']
+        fields = ['pesel', 'address', 'start_date', 'end_date', 'tshirt_size', 'comments']
         labels = {
             'pesel': 'Pesel',
             'address': 'Adres zameldowania',
+            'start_date': 'Data przyjazdu :-)',
+            'end_date': 'Data wyjazdu :-(',
```
Comment (radrow) [inline wwwapp/forms.py]:
```suggestion
            'end_date': 'Data wyjazdu',
```
Remove this emoji pls

File: static/js/moment-locale-pl.js (hunk 1 above)
Comment (radrow) [reply static/js/moment-locale-pl.js]: Seriously, what the fuck? Should we PR this library or whatever it is?

Comment (krzys-h) [reply static/js/moment-locale-pl.js]: https://github.com/moment/moment/blob/develop/locale/pl.js

**Pull request merged**